util/devel/grepstdchdrs

```diff
@@ -18,4 +18,10 @@ fi
 
 filter=${1:+cat}
 cd $project_dir && \
-find compiler \( \( -name '*.cpp' -o -name '*.h' \) -a ! -name 'bison-chapel.cpp' -a ! -name 'flex-chapel.cpp' -a ! -name 'flex-chapel.h' \) -exec egrep ${1:--s} '^[       ]*#[    ]*include[      ][      ]*<[    ]*(assert|ctype|errno|float|iso646|limits|locale|setjmp|signal|stdarg|stddef|stdio|stdlib|string|time|wchar|wctype)[.]h[   ]*>' {} \; -print | `echo "${filter:-grep -v include[^/]}"`a
+find compiler \( \( -name '*.cpp' -o -name '*.h' \) -a \
+  ! -name 'bison-chapel.cpp' -a \
+  ! -name 'flex-chapel.cpp' -a \
+  ! -name 'flex-chapel.h' -a \
+  ! -name 'sys_basic.h' \) \
```

Comment:
Is this line useful?  It looks like find is looking in the compiler directory but the sys_basic.h file you modified is in the runtime directory

Reply:
> since the upcoming PR #17586 will use sys_basic.h in compiler code

PR #17586 adds a symbolic link to that file from the compiler source code